templates/eu-packager-code-3-parts.156x90.tt.svg

```diff
@@ -0,0 +1,18 @@
+<svg width="45px" height="26px" viewBox="0 0 255 152" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Why is it named `.tt.svg` even though it's not processed by Template::Toolkit? You *could* run it through TT to avoid manual string replacement.